src/astProcessor.ts

```diff
@@ -15,11 +15,48 @@ export class AstProcessor {
     }
   }
 
+  private getFeatureFromDoc(doc: any) : Feature {
+    const feature = {
+      name: doc.feature.name,
+      position: doc.feature.location.line,
+      uiElements: this.getUIElementsFromFeature(doc.feature.uiElements)
+    }
+
+    return feature
+  }
+
+  private getUIElementsFromFeature(uiElements: Array<any>) : Array<Element> {
+    let elements = []
+
+    for (let uiElement of uiElements) {
+      const widget = uiElement.items.find(item => item.property === 'type').value.value
```

Comment:
- [ ] Replace the string `'type'` with a constant value or a enum value (the latter may fit better in this case)